mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -155,10 +155,12 @@ def run(self) -> Tuple[Status, Optional[dict]]:
                     self._read_results_file, extra_paths=[temp_dir]))
 
             _LOG.debug("Read data:\n%s", data)
-            if len(data) != 1:
+            if 'metric' in data.columns and 'value' in data.columns:
                 _LOG.warning(
                     "Local run has %d rows: assume long format of (metric, value)", len(data))
                 data = pandas.DataFrame([data.value.to_list()], columns=data.metric.to_list())
+            else:
+                assert 'score' in data.columns
```

Comment:
Please remove that. It's totally ok to not have a `score` data field.

Reply:
Done.